src/battle_controller_player.c

```diff
@@ -886,6 +900,31 @@ static void HandleInputChooseMove(u32 battler)
             gBattlerControllerFuncs[battler] = HandleMoveSwitching;
         }
     }
+    
+    if (sDescriptionSubmenu)
```

Comment:
Shouldn't this be continued as an else if block?

Reply:
That's fair, yeah. I'll update it accordingly